src/OrchardCore.Modules/OrchardCore.Email/Controllers/AdminController.cs

```diff
@@ -59,24 +51,22 @@ public async Task<IActionResult> IndexPost(SmtpSettingsViewModel model)
             {
                 var message = CreateMessageFromViewModel(model);
 
-                if (ModelState.IsValid)
-                {
-                    var result = await _smtpService.SendAsync(message);
+                var result = await _smtpService.SendAsync(message);
```

Comment:
Why you delete the model validation check?

Reply:
did you see  if (ModelState.IsValid) before this?